documentation/api/io.strimzi.api.kafka.model.listener.arraylistener.GenericKafkaListener.adoc

```diff
@@ -0,0 +1,323 @@
+Configures listeners to connect to Kafka brokers within and outside Kubernetes.
+
+You configure the listeners in the `Kafka` resource.
+
+.Example `Kafka` resource showing listener configuration
+[source,yaml,subs="+attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    #...
+    listeners:
+      - name: plain
+        port: 9092
+        type: internal
+        tls: false
+      - name: tls
+        port: 9093
+        type: internal
+        tls: true
+        authentication:
+          type: tls
+      - name: external1
+        port: 9094
+        type: route
+        tls: true
+      - name: external2
+        port: 9095
+        type: ingress
+        tls: false
+        authentication:
+          type: tls
+        configuration:
+          bootstrap:
+            host: bootstrap.myingress.com
+          brokers:
+          - broker: 0
+            host: broker-0.myingress.com
+          - broker: 1
+            host: broker-1.myingress.com
+          - broker: 2
+            host: broker-2.myingress.com
+    #...
+----
+
+[id='property-generic-listeners-{context}']
+=== `listeners`
+You configure Kafka broker listeners using the `listeners` property in the `Kafka` resource.
+Each listener is defined as an array.
```

Comment:
it's not true. The `listeners` field is an array of listener(s) and each listener is an object with some fields like `name`, `port`, and so on.

Reply:
I changed to:

> Listeners are defined as an array.